lib/tasks/thorfile.rb

```diff
@@ -120,38 +120,34 @@ def seed
       puts "[  DONE  ]"
     end
 
+    desc "kit SETUP_PACKAGE", "import files and projects from a specified kit file"
+    method_option :file, :type => :string, :desc => "full path to a zip file"
+    def kit(file)
+      puts "** Importing package..."
+      KitImportJob.perform_now(file: file, logger: default_logger)
+      puts "[  DONE  ]"
+    end
+
     desc "welcome", "adds initial content to the repo for demonstration purposes"
     def welcome
-      # --------------------------------------------------------- Note template
-      if NoteTemplate.pwd.exist?
-        say 'Note templates folder already exists. Skipping.'
-      else
-        template 'note.txt', NoteTemplate.pwd.join('basic_fields.txt')
-      end
-
-      # ----------------------------------------------------------- Methodology
-      if Methodology.pwd.exist?
-        say 'Methodology templates folder already exists. Skipping.'
-      else
-        template 'methodology.xml', Methodology.pwd.join('owasp2017.xml')
+      # zip lib/tasks/welcome_kit
+      directory_to_zip = Rails.root.join('lib', 'tasks', 'welcome_kit')
+      temporary_zip = Tempfile.new
+
+      entries = Dir["#{directory_to_zip}/**/**"] - ['.', '..']
+      Zip::File.open(temporary_zip.path, Zip::File::CREATE) do |zipfile|
```

Comment:
Zipping a file and consequently unzipping it seems clunky to me. Can we have the welcome templates ready as a `.zip` file already? For example, all the templates that we need are found in: `lib/tasks/welcome.zip`.

If that's not possible, another solution is to allow the `setup:kit` task to handle directories instead of a zip file using an option.

Reply:
I did it this way by definition of the task 
> Then setup:welcome task will first zip the contents of lib/task/templates, then call the ported setup:kit to apply them
We had agreed on that somewhere.

The good thing of zipping it is that we have the welcome kit unzipped on `lib/tasks/weilcome_kit`, and its easier to see changes using git.

Reply:
> We had agreed on that somewhere.
I'm late to the party on this. If it is what we decided to do I think it's fine to leave.

I don't understand though- why the repo already has access to all the resources it needs for the kit but can't utilize them in their current form. Creating the kit, and then unpacking the kit does seem like a weird process.

I do get the idea of utilizing the process of kit unpacking because it already exists and is easier to apply. Just seems weird we have all the resources needed to do it already present. If we're committing them, why not also commit the package?